resources/monitoring/values.yaml

```diff
@@ -25,9 +25,13 @@ global:
       slack:
         apiurl: ""
         channel: ""
+        matchExpression:
+          severity: ciritical
```

Comment:
critical... spelling mistake, please fix. And please verify. that all works as before..